### DISCUSSION TITLE
Trouble installing with composer

### DISCUSSION THREAD

**Discussion by YamilBracho** [downforcetech/fluidxml.php#29] [Questions]
Hi.
I am tryoing to install fluid via composer but I got 

[InvalidArgumentException]                                                   
  Could not find package servo/fluidxml at any version matching your PHP vers  
  ion 5.5.38.0 

Which version of php is requiered ?

TIA,
Yamil

**Comment by alepeino** [comment]
As stated in [the composer file](https://github.com/servo-php/fluidxml/blob/master/support/composer.json), the minimum required version is 5.6. The library makes heavy use of variadic functions, which are new to PHP 5.6.

**Comment by YamilBracho** [comment]
Yes!!
My mistake ... I should have read the documentation first... :-)

Thanks a lot!!

**Comment by daniele-orlando** [comment]
Thanks @alepeino for your response.
